master/pservers.html

```diff
@@ -67,8 +66,7 @@ <h2 id="Introduction">Introduction</h2>
   <img
      alt="Image of three types of paint servers."
      src="images/pservers/pservers_example.png"/>
```

Comment:
The png is based on: https://svgwg.org/svg2-draft/images/pservers/pservers_example.svg

Which is already coded in a web-compatible way (the "solid color" example is just a rect with color fill), so it's just a matter of editing out the extras & adjusting the coordinates & re-exporting the PNG. Or just delete the PNG and embed the SVG in the `<img>`, since it would only include well-supported SVG 1.1 features.